.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: JDK ${{ matrix.java_version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        java_version: [1.8]
```

Comment:
Do we care about adding others here eventually? Otherwise it can just be inlined to the action use

Reply:
I wanted to add 11 and 14, but they are failing currently so I removed them for now.